tests/integration/test_debug.py

```diff
@@ -8,7 +8,6 @@
 from typing import List
```

Comment:
Failing tests. Can not use pytest fixture "caplog" with a test that runs in a subprocess

Reply:
Ah darn that's probably because we run our subprocess test cases with `unittest` and not `pytest` 😕

Reply:
I'll update the tests in the removal PR. For now it makes sense to ensure test_debug.py and test_tracer.py maintain their existing behavior.